docs/nodes/mamaki-testnet.md

```diff
@@ -133,7 +133,7 @@ example you can run:
 
 ```sh
 celestia-appd tx staking delegate \
-    celestiavaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd 1000000utia \
+    celestiavaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u4q4gx4p 1000000utia \
```

Comment:
What's the change you are making? You are just modifying key value from the looks of it

Reply:
yes, the one above is from the old binaries and team changed the bech32 format in between, so people get a mismatch error when using this address from the example

Reply:
got it, thank you